internal/scaffold/cmd.go

```diff
@@ -229,8 +229,21 @@ func serveCRMetrics(cfg *rest.Config) error {
 	if err != nil {
 		return err
 	}
-	// To generate metrics in other namespaces, add the values below.
+	// Get the value from WATCH_NAMESPACES
+	watchNamespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		return err
+	}
+
+	// The metrics will be generated from the namespaces which are in ns
+	// NOTE that passing nil or an empty list of namespaces will result in an error.
 	ns := []string{operatorNs}
+
+	// To generate metrics from WATCH_NAMESPACES values if it be as for example ns1,ns2
+	if strings.Contains(watchNamespace, ",") {
```

Comment:
Why not have this logic in the GetWatchNamespace function?

Reply:
Where do you think that this func should be added? Here in the scaffold? Just to encapsulate the logic?

Reply:
Done. Tks for the feedback.

Reply:
Done, it is true we can clean up it. 👍